tools/medtronic-ble-spike/README.md

```
@@ -0,0 +1,97 @@
+
+# Authoritative proof: full JavaSake parity suite (captured 780G trace + PythonSake ciphertexts)
+# plus the spike acceptance gate. 69 tests.
+./gradlew test
+
+# Human-readable narration with a PASS/FAIL verdict and a non-zero exit on any failure.
+./gradlew run
+```
+
+`./gradlew run` prints the handshake stage-by-stage and the SeqCrypt round-trip. Expected tail:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)